packages/components/panel/template.njk

```diff
@@ -1,15 +1,20 @@
 {% set headingLevel = params.headingLevel if params.headingLevel else 3 %}
-{%- if params.label %}
-<div class="nhsuk-panel-with-label
-{%- if params.colour %} nhsuk-panel--{{ params.colour }}{% endif %}
-{%- if params.classes %} {{ params.classes }}{% endif %}"
-{%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
-  <h{{ headingLevel }} class="nhsuk-panel-with-label__label">{{ params.label | safe }}</h{{ headingLevel }}>
-{% else %}
-<div class="nhsuk-panel
-{%- if params.colour %} nhsuk-panel--{{ params.colour }}{% endif %}
-{%- if params.classes %} {{ params.classes }}{% endif %}"
-{%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
-{%- endif %}
+<div class="nhsuk-panel 
+  {% if params.clickablePanel %}clickable-panel {% endif %}
+  {% if params.label %}nhsuk-panel-with-label {% endif %}
```

Comment:
Could do with adding `{%- if params.classes %} {{ params.classes }}{% endif %}` so custom classes can be applied for things like spacing.